deployment/deploy-to-gcp.sh

```diff
@@ -0,0 +1,92 @@
+#! /bin/bash
```

Comment:
Have you tried running this out?

Reply:
yes, the existing deployment in `professionalserviceslab` has deployed by running this script